### ISSUE TITLE
Understanding "for idx in range(10)" in Predict on new data tutorial section

### ISSUE THREAD

**Issue by dhsusf** [jdb78/pytorch-forecasting#825]
I've read some of the closed issues, but I'm still confused about code in the Predict on new data section of the tutorial

new_raw_predictions, new_x = best_tft.predict(new_prediction_data, mode="raw", return_x=True)

for idx in range(10):  # plot 10 examples
    best_tft.plot_prediction(new_x, new_raw_predictions, idx=idx, show_future_observed=False);

I understand our target variable is volume and we're predicting 6 days ahead. I'm a bit confused why we have 10 different predictions. Thanks.

**Comment by eliassz** [comment]
As I can understand there is a lot of groups in that dataset (we have 2 features defines any group ["agency", "sku"]). For every group we make a prediction, but look at 10 of them.

**Comment by dhsusf** [comment]
Thanks. I'm just starting to understand the package better now. I was running the TFT with just one time series and so didn't get the loop, but when I started added more time series it started to make sense. But now, I am a bit confused about the validation set and how long the set actually is.
